Exoft.Gamification.Api/Validators/CreateUserModelValidator.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Exoft.Gamification.Api.Common.Models.User;
+using Exoft.Gamification.Api.Data.Core.Interfaces;
+using FluentValidation;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Exoft.Gamification.Api.Validators
+{
+    public class CreateUserModelValidator : AbstractValidator<CreateUserModel>
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly IRoleRepository _roleRepository;
+
+        public CreateUserModelValidator
+        (
+            IUserRepository userRepository,
+            IRoleRepository roleRepository
+        )
+        {
+            _userRepository = userRepository;
+            _roleRepository = roleRepository;
+
+            RuleFor(user => user.FirstName)
+                .NotEmpty().WithMessage(Resources.ErrorMessages.EmptyField)
```

Comment:
The way you use a resource file is wrong

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/localization?view=aspnetcore-2.2